src/Kalman.jl

```diff
@@ -13,6 +13,10 @@ import GaussianDistributions: ⊕
 meancov(G) = mean(G), cov(G)
 meancov(G::Tuple) = G
 
+symmetrize!(A::Symmetric) = A
+symmetrize!(A) = Symmetric(A)
+symmetrize!(A::Number) = A
```

Comment:
Why is there a `!`? It's not mutating.

Reply:
"M. Schauer house style:" functions marked with ! *may* overwrite. This gives the user the permission to define a modifying `symmetrize!` operation for their own matrix types which might come in handy.